scripts/Engineering/gui/engineering_diffraction/tabs/common/vanadium_corrections.py

```diff
@@ -88,15 +99,19 @@ def _save_correction_files(integration_workspace, integration_path, curves_works
         return
 
 
-def _generate_saved_workspace_file_paths(vanadium_number):
+def _generate_saved_workspace_file_paths(vanadium_number, rb_num=""):
     """
     Generate file paths based on a vanadium run number.
     :param vanadium_number: The number of the vanadium run.
```

Comment:
rb_num should be added to the paramiter list. Also for consistency you should have rb_number or rb_num in both functions